mappings/net/minecraft/block/entity/EndGatewayBlockEntity.mapping

```diff
@@ -1 +1,20 @@
 CLASS net/minecraft/class_2740 net/minecraft/block/entity/EndGatewayBlockEntity
+	FIELD field_12876 GATEWAY_LOGGER Lorg/apache/logging/log4j/Logger;
```

Comment:
```suggestion
	FIELD field_12876 LOGGER Lorg/apache/logging/log4j/Logger;
```
All loggers are usually just named LOGGER or logger since they're only used in a single class.

Reply:
iirc there were collisions... at least enigma didn't allow it

Reply:
ok its fine then